src/main/java/Client.java

```diff
@@ -24,12 +23,17 @@ void Sftp() {
           String hostname = user.getHostname();
           Session session = jsch.getSession(username, hostname, 22);
           session.setPassword(password);
-          session.setConfig("StrictHostKeyChecking", "no");
+
+          Properties config = new Properties();
+          config.put("StrictHostKeyChecking", "no");
+          session.setConfig(config);
+
           System.out.println("Establishing Connection...");
-          session.connect();
+          session.connect(10000);
```

Comment:
This long connection time that is not configurable makes me feel like we won't be able to test if a timeout fails gracefully.
Is it possible to make this a variable we can set so unit tests can run more quickly?

Reply:
@Liriaene Do we want to add the functionality for the user to set the timeout? If so, I can add a setTimeout method that gets called before connect(). Btw apparently the value is in milliseconds

Reply:
Not so much the user, but when we create unit tests if we are forced to wait 10 seconds (10000 milliseconds) for the timeout to occur and we have a unit test that makes sure the program timeout's gracefully, everytime we run unit tests we'll have to wait an additional 10 seconds.

Reply:
@Liriaene So basically just a final long TIMEOUT that we can easily update to change the number of milliseconds for testing? (I'm sure there's a better name than TIMEOUT, but hey, it's early.) 🔨